### ISSUE TITLE
Ability to assign different port for frontend

### ISSUE THREAD

**Issue by bobelev** [pagekite/PyPagekite#47]
Is there any ability to assign one port for frontend-backend communication and another port for listening http clients? It would be very handy to control/firewall traffic.

**Comment by pagekite** [comment]
No, at the moment that is not possible. It's worth considering though, thank you for filing this.
